.gitattributes

```diff
@@ -0,0 +1 @@
+﻿*.d.ts text
```

Comment:
Also if you could pull this single idea into a separate pull request into develop, I think that'd be one small step to breaking up this pull request.